src/_premake_init.lua

```diff
@@ -467,6 +467,18 @@
 	}
 
 	api.register {
+		name = "inlining",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"Default",
+			"Disabled",
+			"Explicit",
+			"Implicit"
```

Comment:
Haha.
Yeah, see it's not really 'implicit' (is it implied by anything in particular?), perhaps it's more like 'automatic', or just 'enabled'?